src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -508,7 +515,11 @@ def _stitchTiles(self, da_list):
         st_data = []
         logging.info("Computing big image out of %d images", len(da_list))
         # TODO: Do this registration step in a separate thread while acquiring
-        das_registered = stitching.register(da_list)
+        try:
+            das_registered = stitching.register(da_list, method=REGISTER_GLOBAL_SHIFT)
+        except ValueError:
+            logging.warning("Registration with global_shift failed, retrying with identity registrar..")
```

Comment:
Please indicate the exception message in the warning. Also change the ".." to either one or three periods.